master/internal/experiment/bulk_action.go

```diff
@@ -246,6 +259,11 @@ func ActivateExperiments(ctx context.Context, system *actor.System,
 func CancelExperiments(ctx context.Context, system *actor.System,
 	experimentIds []int32, filters *apiv1.BulkExperimentFilters,
 ) ([]ExperimentActionResult, error) {
+	if filters != nil {
+		if filters.States == nil {
+			filters.States = runningOrPaused
```

Comment:
i think this check should be `not state in (... list of terminal states)`. e.g., you definitely may want to kill a stopping_completed experiment that is hung trying to shutdown.

Reply:
ok I've been investigating this for a bit. The point of `runningOrPaused` is not to block cancel/kill on other experiments, but to prevent a broad filter query (such as tag=test) from calling Cancel or Kill on experiments where it would be unexpected. I could expand the states, or remove this narrowing filter on filter queries. There doesn't appear to be a good way to get a list of `experimentv1.State` possibilities here and then filter based on whether their equivalent `StateFromProto(state)` is inside `model.TerminalState`